src/core/tests/Test_PatchAssessor.py

```diff
@@ -147,6 +147,19 @@ def test_convert_iso8601_duration_to_total_seconds(self):
         self.assertRaises(Exception, lambda: self.runtime.patch_assessor.convert_iso8601_duration_to_total_seconds('6H5M14S'))
         self.assertRaises(Exception, lambda: self.runtime.patch_assessor.convert_iso8601_duration_to_total_seconds(''))
 
+    def test_write_assessment_perf_logs(self):
+        self.runtime.patch_assessor.stopwatch = Stopwatch(self.runtime.env_layer, self.runtime.telemetry_writer, self.runtime.composite_logger)
```

Comment:
Why do we need to initialize Stopwatch here? Call self.runtime.patch_assessor.start_assessment() that should auto initialize Stopwatch which also validates the initialization part.

Also, you seems like you are validating just the Stopwatch.start() function here. All of Stopwatch's functions should be validated in it's test. Test_PatchAssessor needs to validate the code flow within PatchAssessor.py

For eg: Since we added Stopwatch in start_assesment() in PatchAssessor.py, the test here would be: 
1. Call start_assessment with a Success scenario and a Failure scenario.
2. And in both cases, validate the start, end time and other params within Stopwatch and set as expected

Reply:
Done